kafka/protocol/__init__.py

```diff
@@ -42,4 +42,6 @@
     31: 'DeleteAcls',
     32: 'DescribeConfigs',
     33: 'AlterConfigs',
+    36: 'SaslAuthenticate',
```

Comment:
I saw it was missing, but anyway where is API_KEYS used?

Reply:
It's not used atm.